lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -0,0 +1,100 @@
+"""
+Milestones Transformer
+"""
+
+from django.conf import settings
+
+from edx_proctoring.api import get_attempt_status_summary
+from edx_proctoring.models import ProctoredExamStudentAttemptStatus
+from openedx.core.lib.block_structure.transformer import BlockStructureTransformer, FilteringTransformerMixin
+from util import milestones_helpers
+
+
+class MilestonesTransformer(FilteringTransformerMixin, BlockStructureTransformer):
+    """
+    Exclude proctored exams unless the user is not a verified student or has
```

Comment:
@nasthagiri Can I catch up with you about this change? Want to be sure I understand the change in behavior and reflect it if necessary in the docs.